topics/visualisation/tutorials/jbrowse/tutorial.md

```diff
@@ -0,0 +1,269 @@
+>                        - *"JBrowse Track Type [Advanced]"*: `Canvas Features`
+>                        - In *"JBrowse Contextual Menu options [Advanced]"*:
+>                            - In *"Track Menu"*:
+>                                - {% icon param-repeat %} *"Insert Track Menu"*
+>                                    - *"Menu label"*: `See protein at NCBI`
+>                                    - *"Menu url"*: `https://www.ncbi.nlm.nih.gov/gene?term={locus_tag}%5BGene%20Name%5D`
+>                                    - *"Menu icon"*: `Database`
+>
+>    > ### {% icon tip %} Tip: Templating in Contextual Menu Options
+>    >
+>    > There are more valid values for templating than mentioned in the help. When
+>    > you click on a feature in the JBrowse instance, it will present all of the
+>    > properties of the feature. Any of the top level properties can be used
+>    > directly in your templating
+>    {: .tip}
+>
+> 2. View the output
+>
+>    ![genes in JBrowse](../../images/jbrowse/genes.png "Genes visualised in JBrowse. The lighter coloured genes on the denovo track are coloured based on the confidence score given to the prediction by AUGUSTUS. NCBI genes do not have scores, these are more like an official gene set.")
+>
+> 3. Turn on both tracks of data.
+>
+> 4. Navigate to `21,200`, either manually, or by copying and pasting the location block: `NC_000913.3:18351..24780`
+>
+> 5. Right click on the `yaaY` gene, and click the "See protein at NCBI" menu option.
+>
+>    This menu option is dynamic, try it with a few other features from the `genes (NCBI).gff3` track. These features have a `locus_tag` and the menu button we added will open a URL to an NCBI search page for the value of this `locus_tag` attribute.
+>
+>    ![ncbi view](../../images/jbrowse/ncbi.png "The NCBI iframe within JBrowse, you can see how to link to external databases from JBrowse and help the viewer obtain as much information or context as they might need... but you have to build it yourself.")
+{: .hands_on}
+
+Contextual menus can be used to link to more than just NCBI.
+
+- The links can go anywhere such as web search services (e.g. Google) or genomics web services (e.g. EBI)
+- Some sites use the IFrame action to link genes to local services where users are expected to submit annotation notes or data.
+
+# Sequencing, Coverage, and Variation
+
+This is the next major category of data that people wish to visualize, sequencing, coverage, and variation. The sequencing data can be of any type, and does not necessarily need to be sequencing data, as long as the results are formatted as BAM files. Here we will add several coverage tracks (essentially a line plot along the genome, associating a position with a value) with various visualisation options like scaling and display type. Each of these visualisation options can be useful in different situations, but it largely is a matter of preference or what you are used to seeing.
```

Comment:
wish to visualize, sequencing, coverage, and variation -> wish to visualize: sequencing, coverage, and variation